code/game/turfs/open/floor/misc_floor.dm

```diff
@@ -336,3 +336,13 @@
 /turf/open/floor/material/meat/airless
 	initial_gas_mix = AIRLESS_ATMOS
 	baseturfs = /turf/open/misc/asteroid/airless
+
+/// TGMC EMBLEM ///
```

Comment:
Don't do this ascii stuff, it breaks auto doc and looks ugly. Just a normal `/// TGMC emblem` comment will do

Reply:
Unfortunately this is already on master and just got merged in here 🙃 
I can still delete it though.

Reply:
Oh you shouldnt' have bothered, I hate github showing me merge commits in my notifications